lib/screens/recording_screen.dart

```diff
@@ -169,13 +201,42 @@ class _RecordingScreenState extends State<RecordingScreen> {
         child: Row(children: [
           Text(
             source,
-            style: TextStyle(fontWeight: FontWeight.bold),
+            style: TextStyle(fontWeight: FontWeight.bold, fontSize: 18.0),
           ),
           Expanded(child: Icon(Icons.compare_arrows)),
-          Text(target, style: TextStyle(fontWeight: FontWeight.bold))
+          DropdownButtonHideUnderline(child: _dropDownButton())
         ]));
   }
 
+  Widget _dropDownButton() {
+    return FutureBuilder(
+        future: fetchLanguages("type=local"),
+        builder: (context, snapshot) {
+          if (snapshot.hasError || !snapshot.hasData)
+            return Text(defaultTargetLang);
```

Comment:
defaultTargetLang should be fetched from user preferences.

Reply:
This will be done as the next feature